applications/IgaApplication/tests/single_patch_test/single_patch_project_parameters.json

```diff
@@ -0,0 +1,80 @@
+{
+    "problem_data": {
+        "problem_name": "SinglePatchExample_kratos",
+        "echo_level": 0,
+        "parallel_type": "OpenMP",
+        "start_time": 0,
+        "end_time": 0.1
+    },
+    "solver_settings": {
+        "model_part_name": "IgaModelPart",
+        "domain_size": 1,
+        "echo_level": 0,
+        "buffer_size": 2,
+        "analysis_type": "linear",
+        "model_import_settings": {
+            "input_type": "use_input_model_part"
+        },
+        "material_import_settings": { "materials_filename": "single_patch_test/single_patch_materials.json" },
+        "time_stepping": { "time_step": 1 },
+        "rotation_dofs": false,
+        "reform_dofs_at_each_step": false,
+        "line_search": false,
+        "compute_reactions": true,
+        "block_builder": true,
+        "clear_storage": false,
+        "move_mesh_flag": true,
+        "convergence_criterion": "residual_criterion",
+        "displacement_relative_tolerance": 0.0001,
+        "displacement_absolute_tolerance": 1E-09,
+        "residual_relative_tolerance": 0.0001,
+        "residual_absolute_tolerance": 1E-09,
+        "max_iteration": 1,
+        "solver_type": "static",
+        "linear_solver_settings": {
+            "solver_type": "amgcl",

```

Comment:
do you need amgcl here?

Reply:
no, which one would be recommended?

Reply:
I would not specify it at all and use the solution from the StructuralMechanics Test-Factory